site/gdocs/centered-article.scss

```diff
@@ -23,6 +23,118 @@
             color: $vermillion !important;
         }
     }
+
+    .toc-wrapper {
```

Comment:
A lot of this duplicates the existing site sidebar CSS because much of that CSS is stuck inside media queries (because for old-style entries, the sidebar is always open on xxlg, which we can't do here because it would obscure full-width content beneath it)